broadleaf-thymeleaf3-presentation/src/main/java/org/broadleafcommerce/presentation/thymeleaf3/resolver/BroadleafTemplateResourceUtils.java

```diff
@@ -0,0 +1,115 @@
+package org.broadleafcommerce.presentation.thymeleaf3.resolver;
+
+import org.thymeleaf.util.StringUtils;
+
+final class BroadleafTemplateResourceUtils {
```

Comment:
why do we need this at all ?
I guess if we use delegate it is not required ?
Also I'm not sure if don't have something similar already...